src/source/authentication.rst

```diff
@@ -235,7 +244,15 @@ After obtaining the authorization code at step 2 the client needs to make a
     Host: admarkt.demo.qa-mp.so
     Content-Type: application/x-www-form-urlencoded
 
-    grant_type=authorization_code&code=AUTH_CODE&client_id=YOUR_CLIENT_ID&client_secret=YOUR_CLIENT_SECRET&redirect_uri=https://yoursite.com/code
+    -d 'grant_type=authorization_code&code=AUTH_CODE&client_id=YOUR_CLIENT_ID&client_secret=YOUR_CLIENT_SECRET&rredirect_uri=https://yoursite.com/code' https://admarkt.marktplaats.nl/accounts/oauth/token
+.. warning::
+    Beware of the Content-type: build it as a `multipart/form-data`.
+
+    Beware also that you need to know the `user_id` on behalf of which the Authorization server accesses data. You can
+    obtain this through `passing the proper cookie: <https://pages.github.es.ecg.tools/ecg-icas/api/doc/tenant/api/post_auth_cookie.html>`_
+    from the user's browser (since it's on the same domain as the tenant).
+    Check the `tenant documentation: <https://pages.github.es.ecg.tools/ecg-icas/api/doc/tenant/api/get_config.html>`_ on how to get the `user_id` and which are the relevant cookies to configure for this.
```

Comment:
hold up, this is external documentation referring to our internal documentation? This block shoudl not be here. 
The customer should never know the 'user_id'.

Reply:
then how do we get tenants to be have as sellside API partners?

Reply:
ah, nvm I see your point, will remove it

Reply:
>  You can obtain this through `passing the proper cookie:

this is not the API client actually. referred to as "You"